webpack.config.js

```diff
@@ -29,9 +29,13 @@ function webpackConfig(context) {
             options: {
               babelrc: false,
               presets: [
-                ['es2015', { modules: false }],
+                  ["env", {
+                      targets: {
+                          browsers: ["> 2%"]
```

Comment:
we use `["chrome > 30", "IE 11", "Firefox > 30"]` in cms and backoffice. i'm not sure what is the 2% is doing

Reply:
i think we should change cms & backoffice

Reply:
```
and_chr 62
and_uc 11.4
chrome 62
chrome 61
firefox 56
ie 11
ios_saf 11.0-11.2
op_mini all

Reply:
or do both